### PR TITLE
NVML Quickstart

### PR DESCRIPTION
**Pull request by RamanaReddy8801**
**Ticket**: https://new-relic.atlassian.net/browse/NR-258991
**Description**: Created new NVML quickstart
# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include a Data source and Documentation reference?
- [ ] Are all documentation links publicly accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [ ] Did you check your descriptive content for spelling and grammar errors?
- [ ] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

File: quickstarts/nvml/config.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  ## Why monitor NVML?
+  Monitoring your NVML metrics is crucial to ensure optimal performance, reliability, and efficiency of your GPU-based systems.
+  
+  ## Comprehensive monitoring quickstart for NVML 
+  Monitoring NVML will allow you to effectively track the health and performance of your GPUs, leveraging the capabilities of New Relic for data visualization, alerting, and analysis.
+
+  ## What’s included in this quickstart?
+  New Relic NVML monitoring quickstart provides quality out-of-the-box reporting:
+  - Dashboards (power usage, GPU utilisation, clocks, etc)
+  - Alerts for NVML (GPU temperature, Power usage)
```
Comment (sjyothi54) [inline quickstarts/nvml/config.yml:13]:
```suggestion
  - Alerts for NVML (GPU temperature, power usage)
```

File: quickstarts/nvml/config.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  - New Relic
+  - Ramana Reddy
+title: NVML
+documentation:
+  - name: NVML integration documentation
+    description: |
+      Monitor and instrument your NVML with New Relic to gain deep insights into your performance.
+    url: https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/nvml-integration/
+keywords:
+  - NVIDIA
```
Comment (sjyothi54) [inline quickstarts/nvml/config.yml:30]:
```suggestion
  - NVIDIA
  - NVIDIA Management library
```

**Review by sjyothi54 — CHANGES_REQUESTED** [review]
@RamanaReddy8801 Change the nvml logo to nvidia official logo and work on requested changes

**Review by sjyothi54 — APPROVED** [review]
Looks good

**Review by pkudikyala — APPROVED** [review]
LGTM

**Pull request merged**